### PR TITLE
added link

### PR DESCRIPTION
**Pull request by chriselias**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,6 @@
   <body>
     <h1>Pizza Time</h1>
     <p>this has been updated...</p>
+    <a href="#">hey</a>
```
Comment (deepdishdev) [inline index.html]: Can you add the url here?

Comment (chriselias) [reply index.html]: Yep.

**Comment by deepdishdev** [comment]
ok

**Review by deepdishdev — CHANGES_REQUESTED** [review]
go ahead

**Review by deepdishdev — APPROVED** [review]
looks good

**Pull request merged**